### ISSUE TITLE
Dead link

### ISSUE THREAD

**Issue by agilly** [singularityhub/singularityhub.github.io#202]
Hi,

on the documentation website, there is a link on this page https://singularityhub.github.io/singularityhub-docs/docs/getting-started/ pointing to:

 https://singularityhub.github.io/singularityhub-docs/docs/getting-started/build

, that is dead.

Cheers,

A

**Comment by vsoch** [comment]
Thanks for the pointer! There are indeed some dead links and I can fix this up tomorrow. In the meantime, the markdown files are here for you to browse (without using those same links) https://github.com/singularityhub/singularityhub-docs/tree/master/_docs/getting-started

**Comment by vsoch** [comment]
okay all fixed! The builds link should have linked to https://singularityhub.github.io/singularityhub-docs/docs/builds/. You might need to open devtools / check to not cache the page when it's open to clear the cache, or just wait a little bit or use another browser, but the update is done and (I just checked) the links should be working now. Closing issue, let me know if you have any other questions, and thanks for the heads up!
